client/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class App extends React.Component {
+
+  constructor(props) {
+    super(props);
+    const idxPosProductId = 1;
```

Comment:
My proxy server doesn't store productIds or details--what are you planning to use this state for?

Right now my proxy server bundle is serving up header and footer components, which I believe are static and don't reply on the product.